### ISSUE TITLE
Failing color contrast for default inline code styling for HTML

### ISSUE THREAD

**Issue by batpigandme** [quarto-dev/quarto-cli#2548]
### Bug description

Currently, the default combination of background and foreground colors for inline code is below 4.5. 

This is with the latest build of Quarto (from this git repo). I'm on a Mac, but I don't think that's relevant, as it's just about the HTML styling.

Live page of the qmd file is here <https://mara.quarto.pub/quarto-inline-code/>.

Here is the  [`.qmd` source file](https://github.com/batpigandme/quarto-inline-code/blob/main/quarto-inline-code.qmd) (the contents of which are also shown below).

````
---
title: "quarto-inline-code"
format: html
---

Here is an example of `code` inline. 

The color I'm getting for the inline code with no styling added is `#9954bb`. 
The default background color for inline code is `#f6f6f6`.
The contrast ratio for these two is 4.47 (too low). 
I can make this [WCAG compliant](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast) (CCR >= 4.5 for body text) by increasing the contrast by setting `$code-color` to `#9753b8`, but it would be nice if this was the default behavior.

Here is the [GitHub repo with code for this page](https://github.com/batpigandme/quarto-inline-code).
```` 

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.
